migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc

```diff
@@ -1,8 +1,41 @@
 :_mod-docs-content-type: ASSEMBLY
 [id=ossm-migrating-gateways-assembly]
-= Migrating gateways to Service Mesh 3.0
+= Migrating gateways from Service Mesh 2 to Service Mesh 3
 include::_attributes/common-attributes.adoc[]
 :context: ossm-migrating-gateways-assembly
 
 toc::[]
 
+This section is if you have been using gateways with {SMProductName} {SMv2Version} and are migrating them to {smproduct} 3.0. Migrating gateways between {SMProduct} 2 and {SMProduct} 3 is very similar to migrating regular workloads.
+
+There are two methods for migrating gateways from {SMProductName} 2 to {SMProduct} 3:
+
+* Gateway canary migration
+* Gateway in place migration
+
+Follow the procedure for the scenario that best fits your environment.
+
+include::modules/ossm-migrating-gateways-canary.adoc[leveloffset=+1]
+
+.Next steps
+
+* xref:../done/ossm-migrating-complete-assembly.adoc[Complete your migration]
+
+include::modules/ossm-migrating-gateways-in-place.adoc[leveloffset=+1]
+
+.Next steps
+
+* xref:../done/ossm-migrating-complete-assembly.adoc[Complete your migration]
```

Comment:
```suggestion
* xref:../done/ossm-migrating-complete-assembly.adoc[Completing the Migration]
```

- As per the redirected link, update the topic heading as "Completing the Migration".